engines/chewy/text.cpp

```diff
@@ -181,7 +181,7 @@ Font::~Font() {
 	_fontSurface.free();
 }
 
-::Graphics::Surface *Font::getLine(Common::String text) {
+::Graphics::Surface *Font::getLine(const Common::String text) {
```

Comment:
If this change is in response to my comment, this is not what I meant. The point was to turn this parameters into a reference to avoid a useless copy.